src/JabbR-Core/Properties/Chat.bbcnews.css

```diff
@@ -0,0 +1,26 @@
+﻿.bbc_wrapper .bbc_header
```

Comment:
This was all previously under content/themes/default, lets keep out of properties (how did it end up here?)